lang/main.json

```diff
@@ -395,6 +395,8 @@
         "videoQuality": "Manage call quality"
     },
     "liveStreaming": {
+        "limitNotificationDescriptionWeb": "Due to high demand your recording will be limited to {{limit}} min. For unlimited recordings try <a href={{url}} rel='noopener noreferrer' target='_blank'>{{app}}</a>.",
```

Comment:
Should this say "streaming" instead of "recording"?

Reply:
Fixed!